homeassistant/components/elkm1/sensor.py

```diff
@@ -1,24 +1,29 @@
 """Support for control of ElkM1 sensors."""
 from . import DOMAIN as ELK_DOMAIN, ElkEntity, create_elk_entities
 
+DEPENDENCIES = [ELK_DOMAIN]
```

Comment:
We don't need this anymore with the manifest.json.

Reply:
od